packages/dht/src/dht/store/StoreRpcLocal.ts

```diff
@@ -184,8 +181,7 @@ export class StoreRpcLocal implements IStoreRpc {
 
     // RPC service implementation
     async storeData(request: StoreDataRequest): Promise<StoreDataResponse> {
-        const ttl = Math.min(request.ttl, this.maxTtl)
-        const { key, data, createdAt, creator } = request
+        const { key, data, creator, createdAt, ttl } = request
```

Comment:
Could be useful to allow users to set a max TTL for any data that they store as before. Now the requestor is in control